heroku/cli
# Refactor: migrate apps:unlock command to oclif/core

## Testing
- Check out this branch and run `yarn` and `yarn build`
- Choose an app that you have admin privileges for and check to see if access is locked or unlocked.
- If it is unlocked, run `heroku apps:lock -a APP-NAME`
- Run `./bin/run apps:unlock -a APP-NAME`
    - You should get a message that the app has been unlocked and you should be able to verify this in dashboard
- Run `./bin/run apps:unlock -a APP-NAME` again
    - This time you should get an error message that says the app is not locked.

## Note
This PR also includes changes to a couple of test files. For some reason, my tests were failing because of mocha imports in two other files.
